### DISCUSSION TITLE
Docker crash cannot register gchar

### DISCUSSION THREAD

**Discussion by bols1992** [dlemstra/Magick.NET#1269] [Help]
I currently have the problem that when I create a Docker container and want to use ImageMagick I get the error message

GLib-GObject-WARNING **: 01:39:59.593: cannot register existing type 'gchar
GLib-GObject:ERROR:../gobject/gvaluetypes.c:455:_g_value_types_init: assertion failed: (type == G_TYPE_CHAR)
Bail out! GLib-GObject:ERROR:../gobject/gvaluetypes.c:455:_g_value_types_init: assertion failed: (type == G_TYPE_CHAR)'

The container crashes when I run the application, under windows there are no problems I have looked at the instructions at Crossplatform Support but unfortunately I find the error Not.
I use the current version

maybe someone can help

**Comment by dlemstra** [comment]
Are you using `GtkSharp`? Your error message looks a lot like this one: #1126

**Comment by bols1992** [comment]
I don't use GtkSharp i only add Magick.net Core and Magick.Net Q16 ANY CPU to my Visual Studio Project

**Reply by dlemstra** [reply]
I suspect this is something in your container setup that is causing this. Is it possible to create a **small** example project that demonstrates this issue?

**Reply by bols1992** [reply]
You are Right, i created a new empty project and implement an API Controller for test that worked in Docker

For another component I need the following packages, that crashes the Docker
Does one thing sound familiar to you as to what could be causing this?

RUN apt-get update \
    && apt-get install -y --allow-unauthenticated \
        libc6-dev \
        libgdiplus \
        libx11-dev \
        libharfbuzz0b \
        libfreetype6 \
        fontconfig \
        software-properties-common\
        libicu-dev\
        libfontconfig1\
        libpango-1.0-0\
        libpangocairo-1.0\

**Comment by dlemstra** [comment]
I would suspect that this is caused by `libpango` and `libpangocairo`. They both use `glib` where you are getting this exception from. It might also be possible that the component that requires those packages is causing this.
